setup.py

```diff
@@ -34,7 +34,8 @@
     'sklearn-crfsuite>=0.3.6,<1.0',
     'immutables~=0.9',
     'tensorflow~=1.2; python_version < "3.7"',
-    'tensorflow>=1.13.1,<2.0; python_version >= "3.7"'
+    'tensorflow>=1.13.1,<2.0; python_version >= "3.7"',
+    'ciscosparkapi'
```

Comment:
You should make ciscosparkapi an optional dependency:

```
 extras_require={
        'bot': ['ciscosparkapi'],
    },
```
https://github.com/psf/requests/blob/3c1d36b827417fdeaf5a1c106129de30dac371d7/setup.py#L98

Reply:
Try to compile the library and verify that the following syntax can work:
```
pip install mindmeld[bot]
```

Reply:
Tested it out using `pip install -e .[bot]`
It's working fine.